<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableWidget" name="tableWidget_Datos">
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>nombre</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>apellido</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>edad</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>clase</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_Borrar">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Borrar</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit_Nombre"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Apellido"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Edad"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Clase"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Agregar">
        <property name="text">
         <string>Agregar Usuario</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>765</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
